docs/_articles/en/v3.x/ci-cd-integration.md

```diff
@@ -0,0 +1,32 @@
+---
+title: CI/CD Integration
+lang: en
+redirect_from: /en/ci-cd-integration
+---
+<<<<<<< Updated upstream
+=======
```

Comment:
@teresa-allen-sfdc , you're going to want to remove lines 6 and 7 here.

Reply:
@jfeingold35 Done